msobooking.yaml

```
@@ -0,0 +1,473 @@
+      ol.releasedatetime:
+        type: "ol.releasedatetime"
+        column: "RELDATE"
+      criminaljustice.releasetype:
+        type: "criminaljustice.releasetype"
+        column: "RELTYPE"
+      ol.admissiontype:
+        type: "ol.admissiontype"
+        column: "ADTYPE"
+      ol.commitdatetime:
+        type: "ol.commitdatetime"
+        column: "COMDATE"
+    name: "jailstay"
+
+  person:
+    fqn: "general.person"
+    entitySetName: "MSOInmates"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "PCP"
+      nc.PersonGivenName:
+        type: "nc.PersonGivenName"
+        column: "FIRSTNAM"
+      nc.PersonSurName:
+        type: "nc.PersonSurName"
+        column: "LASTNAME"
+      nc.SSN:
+        type: "nc.SSN"
+        column: "SSNUM"
+      nc.PersonSex:
+        type: "nc.PersonSex"
+        column: "SEX"
+      nc.PersonSuffix:
+        type: "nc.PersonSuffix"
+        column: "AFFIX"
+      nc.PersonBirthDate:
+        type: "nc.PersonBirthDate"
+        column: "BIRTH"
+      nc.PersonEthnicity:
+        type: "nc.PersonEthnicity"
+        column: "HISPANIC"
+      nc.PersonRace:
+        type: "nc.PersonRace"
+        column: "RACE"
+      nc.PersonMiddleName:
+        type: "nc.PersonMiddleName"
+        column: "MIDDLE"
+      ol.datasource:
+        type: "ol.datasource"
+        column: "DATASOURCE"    
+    name: "person"
+
+  fbinumber:
+    fqn: "ol.identification"
+    entitySetName: "MSOFBINumbers"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "FBI"
+      ol.stringnumber:
+        type:  "ol.stringnumber"
+        column: "FBI"
+    name: "fbinumber"
+
+  temporaryrelease:
+    fqn: "ol.temporaryrelease"
+    entitySetName: "MSOTemporaryrelease"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "RELSTDT"
+      ol.datetimestart:
+        type: "ol.datetimestart"
+        column: "RELSTDT"
+      ol.datetimeend:
+        type: "ol.datetimeend"
+        column: "RELRNDT"
+      ol.projectedenddatetime:
+        type: "ol.projectedenddatetime"
+        column: "RELDBDATE"
+    name: "temporaryrelease"
+
+  releaseofficer:
+    fqn: "ol.employee"
+    entitySetName: "MSOOfficer"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "RELOFF"
+      publicsafety.employeeid:
+        type: "publicsafety.employeeid"
+        column: "RELOFF"
+    name: "releaseofficer"
+
+  dnasample:
+    fqn: "ol.medicaltest"
+    entitySetName: "MSODnaSample"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "DNA_SAMPLE_STATUS"
+      ol.status:
+        type: "ol.status"
+        column: "DNA_SAMPLE_STATUS"
+    name: "dnasample"
+
+  sentence:
+    fqn: "ol.sentence"
+    entitySetName: "MSOSentence"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SENTENCE_PK"
+      ol.notes:
+        type: "ol.notes"
```

"Notes" is kind of a vague property so I took a look at what was in this column. I think better in ol.type. Values: 
MINIMUM MAXIMUM SENTENCE

 INDETERMINATE FINE & COSTS ONLY
 NONE

 FLAT SENTENCE
```suggestion
      ol.type:
        type: "ol.type"
```